migrations/000031_Addv1dot5Columns.up.sql

```diff
@@ -0,0 +1,28 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+-- health_authority_id is nullable to preserve backwards compatibility.
+ALTER TABLE exposure
+  ADD COLUMN health_authority_id INT REFERENCES HealthAuthority(id),
```

Comment:
Index this?

Reply:
this isn't in any query at the moment.
will index later if that changes.

This will be used if a key is revised, we'll assert that it is being revised by the same health authority ID.